docs/Makefile

```diff
@@ -21,3 +21,27 @@ help:
 
 upload: html
 		./upload-gh-pages.sh build/html/ pyAFQ yeatmanlab
+
+.PHONY: distclean realclean
+
+# find artifacts from sphinx-gallery in examples folder
+ARTIFACTS := $(strip \
```

Comment:
One other thought: 
- Instead of hardcoding `Makefile` to look for `examples` and `auto_examples` directories, could derive from the `sphinx_gallery_conf.examples_dirs` and `sphinx_gallery_conf.gallery_dirs` respectively in `conf.py`.

Reply:
Added new enhancements request #491